source/includes/fact-ObjectId-construct.rst

```diff
@@ -1,5 +1,10 @@
-- a 4-byte value representing the seconds since the Unix epoch,
+- a 4-byte *timestamp value*, representing the ObjectId's creation,
+  measured in seconds since the Unix epoch
 
-- a 5-byte random value, and
+- a 5-byte *random value*
 
-- a 3-byte counter, starting with a random value.
+- a 3-byte *incrementing counter*, initialized to a random value
+
+While the BSON format itself is little-endian, the *timestamp* and
+*counter* fields are big-endian, with the most significant bytes
```

Comment:
Sorry to dump this here, but I'd suggest changing 'fields' to 'values' - 'fields' implies there are actual fields to grab in the ObjectId JSON, where its actually just addressing that specific chunik of the byte string.

You can make this change on your local branches and force-push to your origin to update the PRs.